src/WrapperApp/components/InputEditor/InputFilesEditor.tsx

```diff
@@ -96,6 +98,11 @@ export function InputFilesEditor(props: InputFilesEditorProps) {
 						return index1 - index2;
 					})
 					.map(([name, value]) => {
+						const isLargeFile = value.length > largeFileSize;
+						const content = !isLargeFile
```

Comment:
What will happen if we have large SOBP data file, it will get truncated as expected and then we submit simulation from "Input files" ? Will it send to backend full or truncated file ?

Reply:
Editor uses its own copy, therefore download and actual generated content is not affected.

Reply:
I confirm this works